lib/live_view_google_maps/application.ex

```diff
@@ -11,6 +11,8 @@ defmodule LiveViewGoogleMaps.Application do
       LiveViewGoogleMapsWeb.Telemetry,
       # Start the PubSub system
       {Phoenix.PubSub, name: LiveViewGoogleMaps.PubSub},
+      # Start the Registry
+     {Registry, keys: :unique, name: :maps},
```

Comment:
when compiling I am getting some warnings:

```
warning: module attribute @impl was not set for function handle_info/2 callback (specified in Phoenix.LiveView). This either means you forgot to add the "@impl true" annotation before the definition or that you are accidentally overriding this callback
  lib/live_view_google_maps_web/live/page_live.ex:26: LiveViewGoogleMapsWeb.PageLive (module)

warning: module attribute @name was set but never used
  lib/maps.server.ex:6

warning: LiveViewGoogleMapsWeb.Maps.new_sighting/3 is undefined or private. Did you mean one of:

      * new_sighting/0
      * new_sighting/1

  lib/maps.server.ex:68: LiveViewGoogleMapsWeb.Server.handle_cast/2

warning: LiveViewGoogleMapsWeb.Maps.subscribe/2 is undefined or private. Did you mean one of:

      * subscribe/0
      * subscribe/1

  lib/live_view_google_maps_web/live/page_live.ex:21: LiveViewGoogleMapsWeb.PageLive.handle_params/3

warning: Map.update/3 is undefined or private. Did you mean one of:

      * update/4

  lib/maps.server.ex:65: LiveViewGoogleMapsWeb.Server.handle_cast/2
```

Could you fix them please? I think some of them might be causing a crash when I open the page:

<img width="1057" alt="Screenshot 2020-09-03 at 16 39 39" src="https://user-images.githubusercontent.com/1597565/92136570-1488a200-ee04-11ea-9392-fcc777f3c194.png">